tasks/set_BDI_mapping_mode.py

```diff
@@ -0,0 +1,70 @@
+from cumulusci.tasks.apex.anon import AnonymousApexTask
+from cumulusci.core.exceptions import TaskOptionsError
+import time
+
+
+class SetBDIMappingMode(AnonymousApexTask):
```

Comment:
This is not actually configured as a task in cumulusci.yml; should the implementation move into BDI_API.py?

Reply:
Its a bit more flexible as a task. Even if it doesn't have a short name, it can be used from other tasks. I don't feel very strongly one way or the other, but it does have that advantage as its own task. Perhaps some future demo or trial configuration would want the org to have Advanced Mapping on when its built. Doing it this way is slightly more future-proof and minimal cost.

Reply:
You convinced me. But why not go ahead and add it to cumulusci.yml so that it can be called on its own from the CLI?

Reply:
Just trying not to make "cci task list" too busy until someone actually nominates a use-case for it...I don't feel strongly about that but I'll note that it isn't the only task that lacks a short-name. For example: DeleteData, AnonApex, ...